src/claimList.js

```diff
@@ -3,49 +3,262 @@ import {
+                            />
+                            <div
+                              style="font-size: 16px; line-height: 24px; font-weight: 700"
+                            >
+                              ${getClaimStatus(claim.status)}
+                            </div>
+                          </div>
+                          ${getUnstyledButtonHtml(
+                            "View details",
+                            getClaimHandler(
+                              claim.seqNum,
+                              claim.type,
+                              claim.status
+                            )
+                          )}
+                        `}
+                  </div>`
+                )
+                .join("")}
+            </div>`
+          : html` <div
+              style="background-color: #fff; border: 1px solid #DFE1E2; border-radius: 4px; padding: 32px; font-size: 16px; line-height: 24px"
+            >
+              <b>No recent claims on file.</b> We haven't received a claim from
+              you in the past 12 months. If you recently applied, expect an
+              update within the next couple weeks.
+            </div>`}
+      </div>
+      ${oldClaims.length > 0
+        ? html` <h2 style="font-size: 22px; line-height: 32px; margin: 0">
+              Older
+            </h2>
+            <hr
+              style="
+                margin: 0;
+                border: none;
+                border-top: 1px solid #dfe1e2;
+                margin-bottom: 8px;
+              "
+            />
+            <div
+              style="text-transform: uppercase; font-size: 13px; line-height: 16px; margin-bottom: 16px"
+            >
+              started more than a year ago
+            </div>
+            ${oldClaims
+              .map(
+                (claim) => html`
+                  <button
+                    style="	background: none;
+                      color: inherit;
+                      font: inherit;
+                      cursor: pointer;
+                      display: flex;
+                      align-items: center;
+                      border: 0.5px solid #A9AEB1;
+                      padding: 16px 32px;
+                      background-color: #fff;
+                      border-radius: 4px;
+                      gap: 8px;
+                      text-align: left;
+                      margin-bottom: 8px;
+                      width: 100%;
+                      justify-content: space-between;
+                      outline-offset: 0.25rem"
+                  >
+                    <div>
+                      <b>Claim for ${getClaimTypeContent(claim.type)}</b>,
+                      started ${getFormattedDate(claim.date)}
+                    </div>
+                    <img src="${ICON_BASE_URL}/navigate_next.svg" alt="" />
+                  </button>
+                `
+              )
+              .join("")}`
+        : ""}
```

Comment:
Looks like there's no `onclick` attribute for the these older claim buttons?

Reply:
Fixed!